### DISCUSSION TITLE
unknown property error in Slabelview while building

### DISCUSSION THREAD

**Discussion by TheDemonHuB** [eclipse-glsp/glsp#913] [Q&A]
Hey i just upgraded theia version from 1.27.2 to 1.34.0 

have facing unknown property error in some components of SLabelView
here is the screenshot of error & implementation 

![image](https://user-images.githubusercontent.com/57061430/218052245-39d03a1f-2113-414d-a89d-3ba2ad67dcaf.png)
![image](https://user-images.githubusercontent.com/57061430/218052285-4c8ccd40-e107-4061-a613-ca88677af2fe.png)

can anyone help ??

**Comment by tortmayr** [comment]
Hi @TheDemonHuB,

this error is not directly related to GLSP. It looks like one of your eslint rules is violated (`react/no-unknown-property`).
Unfortunately I'm not familiar with this ruleset so I'm not sure whether there is a proper fix. However, since you are disabling other eslint rules on certain lines anyways (e.g. `react/jsx-filename-extension`) you could simply do the same for line with the error.

In addition, it seems also be possible to whitelist/ignore certain attributes with additional configuration (https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md#rule-options)
